examples/basic/index.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Basic example of using the mcpd SDK in JavaScript.
+ *
+ * This example demonstrates:
+ * - Connecting to an mcpd daemon
+ * - Listing available servers
+ * - Getting tool schemas
+ * - Dynamically calling tools
+ * - Error handling
+ */
+
+const { McpdClient, McpdError } = require('@mozilla-ai/mcpd');
+
+async function main() {
+  // Initialize the client
+  const client = new McpdClient({
+    apiEndpoint: 'http://localhost:8090',
+    // apiKey: 'your-api-key', // Uncomment if authentication is required
+  });
+
+  try {
+    // List all available servers
+    console.log('Available servers:');
+    const servers = await client.getServers();
+    console.log(servers);
+    console.log();
+
+    // Get health status for all servers
+    console.log('Server health:');
+    const health = await client.getServerHealth();
+    for (const [serverName, serverHealth] of Object.entries(health)) {
```

Comment:
nit: I'm assuming this returns an object where the keys are the server names and the values are an object containing the health status? if so we could rename it to something that encompasses that better like healthResponseByServer / healthGroupedByServer or serverHealthsMap or similar


```suggestion
    const healthByServer = await client.getServerHealth();
    for (const [serverName, serverHealth] of Object.entries(healthByServer)) {
```